web/src/cypress/types/ep.d.ts

```diff
@@ -19,6 +19,7 @@ interface EPOptions {
   description?: string
   repeat?: number
   stepCount?: number
+  favorite?: boolean
```

Comment:
Add `isFavorite` as a required field to the `EP` interface above

Reply:
addressed